index.html

```diff
@@ -97,9 +97,8 @@ <h2>
         <li>A payment request event type {{PaymentRequestEvent}}). A <a>payment
```

Comment:
```suggestion
        <li>A payment request event type ({{PaymentRequestEvent}}). A <a>payment
```

Nit, while you're here :).

(or alternatively, `A payment request event type, {{PaymentRequestEvent}}.` No closing `)`)